src/README.md

```diff
@@ -101,6 +101,44 @@ There is also a file watcher, which monitors the `content` directory and automat
 npm run watch
 ```
 
+## Generating chapter images
```

Comment:
Does the figure markdown need to have the `image` property defined, so the script knows what to name the screenshots? Would be good to document any requirements like that.

Reply:
Added in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2461/commits/81e95147024c68df82f667e210ffe79372132e37